src/graphics/Display.jl

```diff
@@ -0,0 +1,18 @@
+"""Fancy Display methods for IJulia
+"""
+module Display
+export vis
+
+import ..Graphviz
+
+using ...CategoricalAlgebra.Graphs
+
+function vis(g::AbstractGraph)
```

Comment:
My thinking was that people don't necessarily want to see the visual version all of the time. I guess we could hide the "show" behind a dedicated import, but then you couldn't turn it off once you imported it. Or is there another function that I can overload without making it the default?

Reply:
I agree. What if, instead of creating `vis`, we overloaded `to_graphviz` with these methods? It's still pretty easy to type in an interactive setting and it's consistent with other parts of the codebase that use `to_composejl`, `to_tikz`, etc.

Reply:
Sounds good to me!

Reply:
Great! Yes, I think that would make sense.